src/main/scala/cognite/spark/v1/AlphaDataModelInstancesRelation.scala

```diff
@@ -238,12 +238,20 @@ class AlphaDataModelInstanceRelation(
       if (andFilters.isEmpty) EmptyFilter else DSLAndFilter(andFilters)
     }
 
+    val instanceSpaceExternalIdFilter = filters
```

Comment:
I don't really understand what we're doing here or why. Could you add a comment explaining it? Why do we pick the first one, shouldn't it be an error to specify more than one, then?

Reply:
We decided during the collab session that we just go for a simple case, that is taking in account only the first `spaceExternalId` provided and use the space of the model in case nothing is provded, and ignore the rest hehe. I added a comment for it, we can throw an error later if we really have to 😁

Reply:
Alright.